ledger/src/blockstore.rs

```diff
@@ -148,6 +148,13 @@ pub struct Blockstore {
     pub completed_slots_senders: Vec<CompletedSlotsSender>,
     pub lowest_cleanup_slot: Arc<RwLock<Slot>>,
     no_compaction: bool,
+    slot_stats: Arc<Mutex<(u64, HashMap<Slot, SlotStats>)>>,
```

Comment:
can you please add a comment what `u64` is here?
It seems like it is the `timestamp` of last purge. Might also be better to use `std::time::Instant`.

Reply:
It could yea, it's just annoying `Instant` needs a custom Default impl.